src/reducers/profile-view.js

```diff
@@ -962,6 +991,49 @@ export const selectorsForThread = (
             !ProfileData.isNetworkMarker(tm)
         )
     );
+    const getNetworkTracingMarkers = createSelector(
+      getCommittedRangeFilteredTracingMarkers,
+      tracingMarkers =>
+        tracingMarkers.filter(
+          marker => marker.data && marker.data.type === 'Network'
+        )
+    );
+    const getNetworkTiming = createSelector(
+      getNetworkTracingMarkers,
+      MarkerTiming.getMarkerTiming
+    );
+    const getScreenshotMarkersById = createSelector(
+      getRangeFilteredThread,
```

Comment:
There's one problem in using getRangeFilteredThread here: we don't get the screenshot that's immediately before this range.

This is a problem for deeply zoomed-in views, eg this view: we miss the first part, where we should see the screenshot immediately before.

This means that maybe we should always return all screenshots... the good thing is that the structure would be calculated only once, so maybe it could be a direct win ?

Reply:
This will be addressed in the `MarkerTable` processing step follow-up.